src/machine.rs

```diff
@@ -178,9 +181,6 @@ impl<'m> Machine<'m> {
             .stdin(stdin)
             .stdout(stdout)
             .stderr(stderr);
-        if let Some(daemonize_arg) = daemonize_arg {
-            cmd = cmd.arg(daemonize_arg);
-        }
```

Comment:
commit log should say why we want this change. Just "This arg is actually a jailer cmd arg, not a firecracker arg."